config/config.go

```diff
@@ -605,12 +604,6 @@ type StmtSummary struct {
 	HistorySize int `toml:"history-size" json:"history-size"`
 }
 
-// TopSQL is the config for TopSQL.
-type TopSQL struct {
-	// The TopSQL's data receiver address.
-	ReceiverAddress string `toml:"receiver-address" json:"receiver-address"`
```

Comment:
I don't know if this will cause deserialization problems, e.g. an outdated config result in fail to start?